srml/staking/src/tests.rs

```diff
@@ -357,12 +357,12 @@ fn rewards_should_work() {
 		Session::on_initialize(System::block_number());
 		assert_eq!(Staking::current_era(), 0);
 		assert_eq!(Session::current_index(), 1);
-		<Module<Test>>::add_reward_points_to_validator(11, 50);
-		<Module<Test>>::add_reward_points_to_validator(11, 50);
+		<Module<Test>>::add_reward_points(vec![(11, 50)]);
```

Comment:
can't we replace this (and such cases where a test needs to reward everyone) with your `add_reward_points_to_all_elected `?

Reply:
yes but this test test more than just that it test that you can reward two times the same and that a not elected one is no-op